administrator/components/com_admin/sql/updates/mysql/4.1.0-2021-07-16.sql

```diff
@@ -27,24 +29,25 @@ CREATE TABLE IF NOT EXISTS `#__guidedtours` (
   KEY `idx_checked_out` (`checked_out`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 DEFAULT COLLATE=utf8mb4_unicode_ci;
 
+-- --------------------------------------------------------
   
 --
 -- Table structure for table `#__guidedtour_steps`
 --
 
 CREATE TABLE IF NOT EXISTS `#__guidedtour_steps` (
   `id` int NOT NULL AUTO_INCREMENT PRIMARY KEY,
-  `tour_id` int(11) NOT NULL DEFAULT 0,
+  `tour_id` int NOT NULL DEFAULT 0,
   `title` varchar(255) NOT NULL,
   `alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL,
-  `published` tinyint(4) NOT NULL DEFAULT 1,
+  `published` tinyint NOT NULL DEFAULT 1,
```

Comment:
Why do the guided tour steps have a default of 1 for published and the guided tours have a default of 0 for published?